src/bika/lims/browser/analyses/view.py

```diff
@@ -875,6 +875,16 @@ def render_unit(self, unit, css_class=None):
         return "<span class='{css_class}'>{unit}</span>".format(
             unit=unit, css_class=css_class)
 
+    def get_category_title(self, analysis):
+        """Returns the title of the category the analysis is assigned to
+        """
+        obj = api.get_object(analysis)
+        cat = obj.getRawCategory()
```

Comment:
Better would be here the naming `cat_uid` and return immediately if empty.
However, this might be actually just an edge case, because we require all services to have a category...